caffe2/utils/CMakeLists.txt

```diff
@@ -28,23 +21,16 @@ list(APPEND Caffe2_CPU_SRCS
   utils/proto_convert.cc
   utils/proto_utils.cc
   utils/proto_wrap.cc
+  utils/threadpool/pthreadpool-cpp.cc
+  utils/threadpool/ThreadPool.cc
   utils/signal_handler.cc
   utils/smart_tensor_printer.cc
-  utils/string_utils.cc
-  utils/threadpool/ThreadPool.cc)
+  utils/string_utils.cc)
```

Comment:
If we are on a non-mobile build (a mobile build cannot reach here) we still need pthreadpool-cpp.cc since NNPACK, QNNPACK, and XNNPACK are not mobile specific.